include/clients/nrt/KDTreeClient.hpp

```diff
@@ -188,7 +188,7 @@ class KDTreeQuery : public FluidBaseClient, ControlIn, ControlOut
       auto kdtreeptr = get<kTree>().get().lock();
       if (!kdtreeptr)
       {
-        // c.reportError("No FluidKDTree found");
+        // c.reportError("No FluidKDTree found"); //why are both this and line 197+214 commented?
```

Comment:
Because we'd very much like to be able to uncomment it one day and report errors from the RT objects.